module/VuFind/src/VuFind/Hierarchy/TreeDataFormatter/AbstractBase.php

```diff
@@ -77,6 +77,23 @@ abstract class AbstractBase implements \VuFind\I18n\HasSorterInterface
      */
     protected $count = 0;
 
+    /**
+     * Throw an exception if hierarchy parent and sequence data is out of sync?
+     *
+     * @var bool
+     */
+    protected $validateHierarchySequences;
+
+    /**
+     * Constructor
+     *
+     * @param bool $validateHierarchySequences Throw an exception if hierarchy parent and sequence data is out of sync?
+     */
+    public function __construct($validateHierarchySequences)
```

Comment:
Might be good to provide a default value here for greater backward compatibility.